microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -3,10 +3,9 @@ title: Detect Windows 11 using User-Agent Client Hints
 description: How to differentiate Windows 10 and Windows 11 using User-Agent Client Hints.
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 01/04/2022
 ms.topic: conceptual
 ms.prod: microsoft-edge
-keywords: microsoft edge, compatibility, web platform, user-agent string, ua string, user-agent client hints, user agent client hints, ua client hints, ua ch, windows 11, win11, detecting windows 11, windows detection
+ms.date: 01/04/2022
```

Comment:
fyi - in 2 files, I think, I moved Date field to bottom, like in DCG's & Doc Auth Pak's templates.